lib/verdict/railtie.rb

```diff
@@ -12,6 +12,7 @@ class Verdict::Railtie < Rails::Railtie
   config.to_prepare do
     # Clear Verdict's cache in order to avoid "A copy of ... has been removed from the module tree but is still active!"
     Verdict.clear_repository_cache
+    Verdict.discovery
```

Comment:
Do we need to discovery on every request in development and test?

Reply:
I believe so, since `Verdict.clear_repository_cache` will clear the repository each time `discovery` needs to be called to repopulate it. discovery cannot be called in the initializer only because this will run before the config.to_prepare and the repository cache will get cleared. let me know if you have an alternative suggestion :)

Reply:
Hmmm, this seems expensive, but nothing we can do in this PR.